cli/lib/tasks/download.js

```diff
@@ -304,7 +313,9 @@ const downloadFromUrl = ({ url, downloadDestination, progress, ca }) => {
  * @param [string] downloadDestination Local filename to save as
  */
 const start = (opts) => {
-  let { version, downloadDestination, progress } = opts
+  let { version, downloadDestination, progress, redirectTTL: passedInTTL } = opts
+
+  const envTTL = util.getEnv('CYPRESS_DOWNLOAD_REDIRECT_TTL')
```

Comment:
Let's remove the ENV for the first release. The redirect count wasn't previously configurable and I'd like to wait for user feedback before adding more config options.

Reply:
Sure it was... couldn't you pass an option to agentOptions that would indicate the max redirects? (in the object its at request._redirect.maxRedirects if you ignore encapsulation constraints)  And wouldn't it be easier to just leave it as an undocumented/stealth feature, that could be removed if it offended rather than going through a whole nother release cycle to add an eminently predictable configuration point?  At least this way, if you went in and looked at the code you'd be like, OH! I can set that environment variable - instead of having to comprehend anything about the code and how it works in order to change it...

Reply:
Yes, it can be configured in @cypress/request, by passing an option into reqOptions, but it wasn't _user_ configurable, unless I'm miss reading something.

I'm not sure I agree that this is a predictable configuration point, it hasn't been a request up to this point. When it comes to configuration, I'm a minimalist. The more configuration you add, the more variables you have for debugging and opportunities for regressions. If I'm wrong and a request comes in to configure this in the future, I'll happily add and document this config option myself. It wouldn't take more than a day's worth of effort and we release bi-weekly.

Reply:
Very well. Made it so.